index.bs

```diff
@@ -1959,6 +2005,21 @@ However, it's often useful to explain the purpose of the algorithm in prose
 callback per toplevel browsing context")
 so that readers can quickly decide whether they need to read the steps in detail.
 
+<h4 id="use-flags-for-states">Use explicit flags for state</h4>
+
+Describing state using explicit flags makes behavior clearer.
```

Comment:
This could use an example, I'm not sure what this means.

Reply:
I've added a TODO to add examples here.